src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -11924,6 +11924,54 @@ HRESULT CEEJitInfo::getMethodBlockCounts (
     return hr;
 }
 
+CORINFO_CLASS_HANDLE CEEJitInfo::getLikelyClass(
+                     CORINFO_METHOD_HANDLE ftnHnd,
+                     CORINFO_CLASS_HANDLE  baseHnd,
+                     UINT32                ilOffset,
+                     UINT32 *              pLikelihood,
+                     UINT32 *              pNumberOfClasses
+)
+{
+    CONTRACTL {
+        THROWS;
+        GC_TRIGGERS;
+        MODE_PREEMPTIVE;
+    } CONTRACTL_END;
+
+    CORINFO_CLASS_HANDLE result = NULL;
+    *pLikelihood = 0;
+    *pNumberOfClasses = 0;
+
+    JIT_TO_EE_TRANSITION();
+
+#ifdef FEATURE_PGO
+
+    // If TieredPGO is enabled, query the per call site class profile.
```

Comment:
The comment says "If TieredPGO is enabled," but there is no check for TieredPGO in this function.

Reply:
I'll reword that; if TieredPGO is not enabled the pgo manager won't find any class profiles.